core/src/main/java/hudson/node_monitors/SwapSpaceMonitor.java

```diff
@@ -85,6 +87,7 @@ public String getColumnCaption() {
      *      use injection
      */
     @Deprecated
+    @Restricted(NoExternalUse.class)
     public static /*almost final*/ AbstractNodeMonitorDescriptor<MemoryUsage> DESCRIPTOR;
```

Comment:
This is currently used in [Alibaba Cloud ECS](https://github.com/jenkinsci/alibabacloud-ecs-plugin/blob/7b287c7752990c9caae62d2f9909b65f2092095e/src/main/java/com/alibabacloud/jenkins/ecs/monitor/FreeMemMonitor.java#L66). That being said, I think it's OK to mark it as restricted as long as we also file a corresponding PR to update the usage in Alibaba Cloud ECS.